src/main/java/uk/gov/hmcts/reform/sscs/model/DescendantCaseData.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+
+import java.util.Map;
+
+@EqualsAndHashCode(callSuper = true)
+@Builder (builderMethodName = "DescendantCaseDataBuilder")
```

Comment:
We don't strictly need to provide a builder name here.. unless you have a preference?

Reply:
This is needed to get a handle on the child.